modules/hostap/src/supp_events.c

```diff
@@ -38,7 +174,6 @@ int generate_supp_state_event(const char *ifname, enum net_event_wpa_supp_cmd ev
 {
 	/* TODO: Replace device_get_binding. */
 	const struct device *dev = device_get_binding(ifname);
-
```

Comment:
unncessary whitespace, if you want to fix the warning that empty line after declarations then `iface` also needs to be cubbed here.

Reply:
This is removal of unnecessary whitespace.

Reply:
Can you please fix this in another commit? The whitespace here is needed but we need club declarations together before that.

Reply:
Actually, this was an inadvertent change, reverting this bit.